src/ResourceManager/ServiceBus/Commands.ServiceBus/Cmdlets/Rule/RemoveAzureServiceBusRule.cs

```diff
@@ -0,0 +1,72 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+namespace Microsoft.Azure.Commands.ServiceBus.Commands.Rule
+{
+    /// <summary>
+    /// 'Remove-AzureRmServiceBusRule' Cmdlet removes the specified Rule
+    /// </summary>
+    [Cmdlet(VerbsCommon.Remove, ServicebusRuleVerb, SupportsShouldProcess = true)]
+    public class RemoveAzureRmServiceBusRule : AzureServiceBusCmdletBase
+    {
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 0,
+            HelpMessage = "The name of the resource group")]
+        [Alias("ResourceGroup")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 1,
+            HelpMessage = "Namespace Name.")]
+        [Alias(AliasNamespaceName)]
+        [ValidateNotNullOrEmpty]
+        public string Namespace { get; set; }
+
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 2,
+            HelpMessage = "Topic Name.")]
+        [Alias(AliasTopicName)]
+        [ValidateNotNullOrEmpty]
+        public string Topic { get; set; }
+
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 3,
+            HelpMessage = "Subscription Name.")]
+        [Alias(AliasSubscriptionName)]
+        [ValidateNotNullOrEmpty]
+        public string Subscription { get; set; }
+
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 4,
+            HelpMessage = "Rule Name.")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        public override void ExecuteCmdlet()
+        {
+            // delete a Rule 
+            if (ShouldProcess(target: Name, action: string.Format(Resources.RemoveRule, Name, Subscription, Namespace)))
+            {
+                WriteObject(Client.DeleteRule(ResourceGroupName, Namespace, Topic, Subscription, Name));
```

Comment:
@v-Ajnava other Remove-* cmdlets were using `PassThru`. Why not for this cmdlet as well?

Also, we should consider whether or not we want to use an `InputObject` parameter set.